docs/se/guides/tracing.adoc

```diff
@@ -262,17 +279,24 @@ cd helidon-quickstart-se-2
 ----
 
 [source,xml]
-.Add the following dependency to `pom.xml`:
-----
-<dependency>
-    <groupId>io.helidon.tracing</groupId>
-    <artifactId>helidon-tracing</artifactId>
-</dependency>
-<dependency>
-    <groupId>io.helidon.tracing.providers</groupId>
-    <artifactId>helidon-tracing-providers-jaeger</artifactId>
-</dependency>
-----
+.Add the following dependencies to `pom.xml`:
+----
+ <dependency>
+     <groupId>io.helidon.tracing</groupId>
+     <artifactId>helidon-tracing</artifactId>   <1>
+ </dependency>
+ <dependency>
+     <groupId>io.helidon.webserver.observe</groupId>
+     <artifactId>helidon-webserver-observe-tracing</artifactId> <2>
+ </dependency>
+ <dependency>
+     <groupId>io.helidon.tracing.providers</groupId>
+     <artifactId>helidon-tracing-providers-jaeger</artifactId>  <3>
+ </dependency>
+----
+<1> Helidon Tracing dependencies.
```

Comment:
All three are Helidon tracing dependencies so this description does not really distinguish this one from the others. Maybe change this first one to something like "Helidon tracing API" instead?

Reply:
ok :)